src/geojson/geojson.ml

```diff
@@ -58,6 +58,22 @@ module Make (J : Geojson_intf.Json) = struct
       let to_json arr = J.array J.float arr
     end
 
+    (* Returns the float array of coordinates if all goes well for any GeoJson type *)
```

Comment:
This looks good to me 👍

Reply:
> This looks good to me 👍

I appreciate your feedback.
Means a lot to me. Thank you😊😊